src/pages/Delivery/Delivery.styles.js

```diff
@@ -0,0 +1,7 @@
+export const styleDelivery = {
+  display: "flex",
+  flexDirection: "column",
+  margin: "3rem auto",
+  width: "1152px",
```

Comment:
Also the width shouldn't be fixed. I suggest it to be `max-width` so that the header can shrink and grow

Reply:
good